src/daft-functions/Cargo.toml

```diff
@@ -23,6 +23,7 @@ python = [
   "common-error/python",
   "daft-core/python",
   "daft-io/python",
+  "daft-dsl/python",
```

Comment:
What's the purpose of this import?

Reply:
i think it's just something we missed originally, so it was just a drive-by to fix it.